frontend/src/libs/components/meditation-tracks-list/meditation-track/meditation-track.tsx

```diff
@@ -0,0 +1,39 @@
+import { Icon, Link } from '#libs/components/components.js';
+import { IconNameToIcon } from '#libs/enums/enums.js';
+
+import styles from './styles.module.scss';
+
+type Track = {
+  id: number;
+  title: string;
+  duration: string;
+  link: string;
+  imageLink: string;
+};
+
+type Properties = {
+  track: Track;
+};
+
+const MeditationTrack: React.FC<Properties> = ({ track }) => {
+  return (
+    <div className={styles['track']}>
+      <img
+        src={track.imageLink}
+        alt="Meditation track background"
+        className={styles['background-image']}
+      />
+      <div className={styles['content']}>
+        <div className={styles['info']}>
+          <h1 className={styles['title']}>{track.title}</h1>
+          <p className={styles['duration']}>{track.duration}</p>
+        </div>
+        <Link to="/" className={styles['play-button'] ?? ''}>
```

Comment:
Whenether styles['play-button'] can be null or undefined, it's not a variable?

Reply:
no, but you can use `as string`

Reply:
fixed